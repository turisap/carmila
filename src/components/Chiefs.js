/**
 * Created by HP on 09-Dec-17.
 */
import React from 'react';

const Chiefs = () => (
    <div>
        <p>Chiefs page</p>
    </div>
);

export default Chiefs;