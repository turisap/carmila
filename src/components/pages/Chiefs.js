/**
 * Created by HP on 09-Dec-17.
 */
import React from 'react';

const Chiefs = () => (
    <div className="container">
        <div className="row">
            <div className="col-md-6">

            </div>
        </div>
    </div>
);

export default Chiefs;