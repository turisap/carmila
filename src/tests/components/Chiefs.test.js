/**
 * Created by HP on 15-Dec-17.
 */
import React from 'react';
import { shallow } from 'enzyme';
import Chiefs from '../../components/pages/Chiefs'

test('should render Chiefs page', () => {
    const wrapper = shallow(<Chiefs/>);
    expect(wrapper).toMatchSnapshot();
});
/**
 * Created by HP on 15-Dec-17.
 */
