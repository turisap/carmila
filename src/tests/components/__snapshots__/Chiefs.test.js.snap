// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Chiefs page 1`] = `
<div>
  <p>
    Chiefs page
  </p>
</div>
`;
